src/include/sof/ipc/common.h

```diff
@@ -65,6 +65,9 @@ struct ipc {
 	/* processing task */
 	struct task ipc_task;
 
+	/* task for flushing outbound messages */
```

Comment:
to the commit message: "the platform driver **returns** -EBUSY?"

Reply:
Thanks, fixed in #6243